packages/braid-design-system/src/lib/components/private/Modal/Modal.tsx

```diff
@@ -221,13 +226,12 @@ export const Modal = ({
           position="fixed"
           inset={0}
           zIndex="modalBackdrop"
-          transition={position === 'center' ? 'fast' : undefined}
           opacity={state !== OPEN ? 0 : undefined}
           pointerEvents={state === CLOSING ? 'none' : undefined}
           className={[
-            styles.backdrop,
-            position === 'left' ||
-              (position === 'right' && styles.horiztontalTransition),
+            (state === OPEN || state === OPENING) && styles.backdrop,
```

Comment:
Not sure the backdrop styling should be conditional. I think the possible improvement here is triggering the opacity change (line 229 above) during the opening phase. Thoughts?